src/uct/ib/rdmacm/rdmacm_listener.c

```diff
@@ -11,6 +11,25 @@
 #include "rdmacm_listener.h"
 
 
+#define UCS_RDMACM_MAX_BACKLOG_PATH        "/proc/sys/net/rdma_ucm/max_backlog"
+
+
+static long ucs_rdmacm_max_backlog()
+{
+    long max_backlog;
```

Comment:
```
{
    static long max_backlog = 0;

    if (max_backlog != 0) {
        return max_backlog 
    }

    ...
}
```
to not read the file each time when this function is called, i.e. use the same approach as in `ucs_socket_max_conn()`

Reply:
you mean in case there is more than 1 rdmacm_listener?

Reply:
on other hand, read the file every time could adjust the parameter in runtime for new listeners without restart server process.

Reply:
i think better read only once, we are doing same way in ucs_socket_max_conn()